backend/src/data/models/profession.ts

```diff
@@ -0,0 +1,33 @@
+import { Sequelize, DataTypes, Model, ModelCtor } from 'sequelize';
+import { ModelName, TableName, ProfessionName } from '~/common/enums';
+import { IProfession } from '~/common/interfaces';
+
+interface ProfessionInstance extends IProfession, Model { }
+
+const createProfessionModel = (orm: Sequelize): ModelCtor<ProfessionInstance> => {
+  const Profession = orm.define<ProfessionInstance>(
+    ModelName.PROFESSION,
+    {
+      name: {
```

Comment:
```suggestion
     [ProfessionKey.NAME]: {
```
You can use enum-keys for all models 🙂